lib/src/revset.rs

```diff
@@ -1141,6 +1141,7 @@ fn try_transform_expression_bottom_up(
         .map(Rc::new))
     }
 
+    #[allow(clippy::type_complexity)]
```

Comment:
Nit: this belongs to the previous commit.

Reply:
Thanks, fixed.